2018/05/25 14:38:50 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "10.39.30.76"
2018/05/25 14:38:50 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "10.39.30.76", referrer: "http://10.39.30.76/test"
2018/05/25 14:39:14 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "10.39.30.76"
2018/05/25 14:39:14 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "10.39.30.76", referrer: "http://10.39.30.76/test"
2018/05/25 14:39:14 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "10.39.30.76"
2018/05/25 14:39:15 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "10.39.30.76", referrer: "http://10.39.30.76/test"
2018/05/25 14:39:16 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "10.39.30.76"
2018/05/25 14:39:16 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "10.39.30.76", referrer: "http://10.39.30.76/test"
2018/05/25 14:39:17 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "10.39.30.76"
2018/05/25 14:39:17 [error] 7#7: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "10.39.30.76", referrer: "http://10.39.30.76/test"
2018/05/25 14:45:13 [error] 7#7: *49 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "m.homework.test"
2018/05/25 14:45:13 [error] 7#7: *49 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "m.homework.test", referrer: "http://m.homework.test/test"
2018/05/25 14:48:35 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "m.homework.test"
2018/05/25 14:48:35 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "m.homework.test", referrer: "http://m.homework.test/test"
2018/05/25 14:48:38 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /test HTTP/1.1", upstream: "http://172.18.0.4:8080/test", host: "m.homework.test"
2018/05/25 14:48:38 [error] 7#7: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: l.blog.server, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.4:8080/favicon.ico", host: "m.homework.test", referrer: "http://m.homework.test/test"
2018/05/25 14:48:49 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/var/log/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 10.39.30.76, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "10.39.30.76"
2018/05/25 14:49:17 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/var/log/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 10.39.30.76, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "10.39.30.76"
2018/05/25 14:50:09 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/var/log/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 10.39.30.76, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "10.39.30.76"
2018/05/25 14:52:44 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/var/log/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 10.39.30.76, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "10.39.30.76"
2018/05/25 16:22:58 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/var/log/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 10.39.30.76, request: "GET /test HTTP/1.1", upstream: "fastcgi://172.18.0.4:9000", host: "10.39.30.76"
2021/11/11 06:17:43 [error] 30#30: *2 open() "/etc/nginx/./develop404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:22:29 [error] 30#30: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:22:45 [error] 30#30: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:23:38 [error] 30#30: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:23:39 [error] 30#30: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:23:49 [error] 30#30: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:26:08 [error] 31#31: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 06:26:12 [error] 31#31: *2 open() "/data404" failed (2: No such file or directory), client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/11 07:53:43 [error] 30#30: *1 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "localhost"
2021/11/11 07:53:57 [error] 30#30: *2 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 172.22.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "localhost"
2021/11/12 01:56:33 [error] 30#30: *1 directory index of "/develop/" is forbidden, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "localhost"
2021/11/12 01:56:34 [error] 30#30: *1 rewrite or internal redirection cycle while internally redirecting to "/index.php", client: 172.24.0.1, server: dev.qms.server, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2021/11/12 01:57:18 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:19 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:20 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:21 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:22 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:22 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:22 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/12 01:57:23 [error] 30#30: *2 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file in /develop/index.php on line 1" while reading response header from upstream, client: 172.24.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "localhost"
2021/11/16 06:44:07 [error] 30#30: *2 open() "/develop/qms/server/php/public404" failed (2: No such file or directory), client: 172.18.0.1, server: dev.qms.server, request: "GET / HTTP/1.1", host: "dev.qms.server"
2021/11/16 06:44:09 [error] 30#30: *2 open() "/develop/qms/server/php/public404" failed (2: No such file or directory), client: 172.18.0.1, server: dev.qms.server, request: "GET /favicon.ico HTTP/1.1", host: "dev.qms.server", referrer: "http://dev.qms.server/"
2021/11/16 06:49:51 [error] 30#30: *1 rewrite or internal redirection cycle while internally redirecting to "/admin/index.html", client: 172.18.0.1, server: dev.qms.server, request: "GET /admin HTTP/1.1", host: "dev.qms.server"
2021/11/17 01:45:35 [error] 30#30: *1 rewrite or internal redirection cycle while internally redirecting to "/admin/index.html", client: 172.20.0.1, server: dev.qms.server, request: "GET /admin HTTP/1.1", host: "dev.qms.server"
